P5/Source/Specs/event.xml

```diff
@@ -8,18 +8,17 @@
 -->
 <?xml-model href="https://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
 <elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="namesdates" ident="event">
-  <gloss xml:lang="en" versionDate="2008-12-09">event</gloss>
+  <gloss versionDate="2008-12-09" xml:lang="en">event</gloss>
+  <gloss versionDate="2023-05-07" xml:lang="de">Ereignis</gloss>
   <gloss versionDate="2008-12-09" xml:lang="fr">évènement</gloss>
-  <desc versionDate="2007-08-01" xml:lang="en">contains data relating to any kind of significant event associated with a person, place, or organization.</desc>
-  <desc versionDate="2007-12-20" xml:lang="ko">개인, 장소, 또는 조직과 관련된 중요한 사건에 해당하는 데이터를 포함한다.</desc>
+  <desc versionDate="2023-05-07" xml:lang="en">contains data relating to any kind of significant event associated with a person, organization, place, object, or other event.</desc>
```

Comment:
Council F2F in Paderborn: Revise to simplify:
"contains data relating to anything of significance that happens."